lib/rubocop/cop/lint/unreachable_code.rb

```diff
@@ -74,11 +74,19 @@ def flow_expression?(node)
         end
 
         def check_if(node)
+          return true if check_literal_if(node)
```

Comment:
It seems this doesn't check whether the outcome of the `if` is a `flow_expression?` (return, exit, etc.) so you would get a false positive for something like:

```ruby
foo if true
bar              # wrongly marked as unreachable
```

Reply:
I missed that, good catch! I've fixed that and added a test case to ensure it doesn't get missed again.